src/main/java/jp/co/future/uroborosql/context/SqlContextImpl.java

```diff
@@ -182,7 +182,7 @@ public String getExecutableSql() {
 			if (executableSql.length() > 0) {
 				// where句の直後に来るANDやORの除去
 				StringBuffer buff = new StringBuffer();
-				if (executableSql.indexOf("WHERE") >= 0 || executableSql.indexOf("where") >= 0) {
+				if (executableSql.toString().toUpperCase().indexOf("WHERE") >= 0) {
```

Comment:
@HidekiSugimoto189 

```java
if (executableSql.toString().toUpperCase().indexOf("WHERE") >= 0) {
```

I think the above implementation is smart when written as follows.

```java
if (executableSql.toString().toUpperCase().contains("WHERE")) {
```